field_utils/FieldCondensedArray.F90

```diff
@@ -1,7 +1,7 @@
 #include "MAPL_Generic.h"
 module mapl3g_FieldCondensedArray
    use, intrinsic :: iso_c_binding, only: c_ptr, c_f_pointer
-   use mapl3g_FieldCondensedArray_private, only: ARRAY_RANK, get_shape => get_fptr_shape
+   use mapl3g_FieldCondensedArray_private, only: ARRAY_RANK, get_fptr_shape_private
```

Comment:
This change really does not accomplish anything.   Before it was renamed on the USE statement.  Now it is renamed manually here.  And the `_private` suffix suggests that it should not be used outside the module (except for testing).

I recommend undoing it.  But I defer to you.

Reply:
I thought about using the generic function via, but it only included the public `get_fptr_shape` specific function. `get_fptr_shape_private` does not act on `ESMF_Field` objects.